src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -211,6 +212,7 @@ export const ChartNote = styled(Box)`
 export const SavedQueryAction = styled(Link)`
   color: ${p => p.theme.gray6};
   margin-left: ${space(2)};
+  display: flex;
```

Comment:
What does the flex here do?

Reply:
The icon had a height of 22px, which is 2 pixels taler than the natural height (20px) of the header container. This caused a very subtle movement when adding the icon.

Adding `display: flex` caused the child element to snap down to the natural height of the parent flex container. TBH I'm not 100% sure _why_ this works but it has worked for me in the past lol.

Reply:
I think the reason this works is the same as above. With the additional `<a>` the line-height doubles up. Setting the `<a>` to `display: flex` removes the line-height. Another fix would be to set `line-height: 0` though that feels a little hacky to me.

Reply:
Hmm, this is strange, the icon actually renders at a height of 16px for me, not 24. Where does that 16px even come from?

Reply:
the containing <a> is what gets the height of 24 for me. the svg does keep the 16px which I believe is the 1em value inherited from `inlineSvg`